paasta_tools/cli/cmds/validate.py

```diff
@@ -712,15 +712,19 @@ def validate_min_max_instances(service_path):
     return returncode
 
 
-def check_secrets_for_instance(instance_config_dict, soa_dir, service_path, vault_env):
+def check_secrets_for_instance(instance_config_dict, soa_dir, service, vault_env):
```

Comment:
```suggestion
def check_secrets_for_instance(instance_config_dict: InstanceConfigDict, soa_dir: str, service: str, vault_env: str):
```
might be nice to add types to functions that we're updating in this file 

(feel free to drop tho)